libexec/basher-_clone

```diff
@@ -5,18 +5,26 @@
 
 set -e
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -lt 1 ]; then
   basher-help _clone
   exit 1
 fi
 
-package="$1"
+package=$1
```

Comment:
Assignments in bash don't do word splitting on variable expansions, quotes are only ever necessary if you use literal whitespace.

http://mywiki.wooledge.org/WordSplitting#Notes